core/timeline_inc.php

```diff
@@ -82,8 +105,9 @@
 		timeline_print_events( $t_events );
 		echo '<div class="widget-toolbox">';
 		echo '<div class="btn-toolbar">';
-		echo '<a class="btn btn-primary btn-sm btn-white btn-round" href="my_view_page.php?days='
-			. $f_days . '&amp;all=1">' . lang_get( 'timeline_more' ) . '</a>';
+		$t_url_params['all'] = 1;
+		$t_href = $t_url_page . '?' . helper_url_query_string( $t_url_params );
```

Comment:
tks, must be replaced with "http_build_query"
Do you have time to provide the change?, i cant until tomorrow at least...